docs/build-from-scratch.md

```diff
@@ -26,6 +28,9 @@ Where `go` is your [import path][4] for Go.
 
 For Go development, it is recommended to add the Go import path (`$HOME/go` in this example) to your path.
 
+### GitHub Release Source Code Archive
+Download the corresponding "Source code" archive from the release artifacts and
```

Comment:
No need to put source code in quotes. It'd be great to add a link to the release page under the word `release`.

Reply:
I used quotes because "Source code" is the name github automatically gives the archive, and I was trying to call attention that specific term. Would you still prefer no quotes?

Reply:
Ah, gotcha. You may bold the words, **Source code**. Better yet: 

Download the archive named `Source code` from the release page.

Reply:
Done